modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/FrontendDataSet.tsx

```diff
@@ -1209,4 +1229,130 @@ const FrontendDataSet = ({
 	);
 };
 
+const FrontendDataSet = ({
+	actionParameterName,
```

Comment:
Is is possible to have here just `props`, and then below
```js
<FrontendDataSetContent
    ...props
```
The big destructuring would then be in `FrontendDataSetContent` component.

Reply:
done